client/src/commands/UserInputUtil.ts

```diff
@@ -208,8 +208,10 @@ export class UserInputUtil {
             });
         }
 
+        quickPickItems.push({label: 'Install + Instantiate new smart contract from package', data: {chaincode: '', version: ''}});
```

Comment:
i think this should say `Install and Instantiate a new smart contract from package....`